lib/ansible/modules/monitoring/datadog_monitor.py

```diff
@@ -122,6 +122,11 @@
         required: false
         default: null
         version_added: "2.3"
+    evaluation_delay:
+        description: ["The number of second to delay data evaluation."]
+        required: false
+        default: null
+        version_added: "2.5"
```

Comment:
This would need to be bumped to `2.6` and get the change in soon, dev freeze is tomorrow.